packages/devtools/test/utils_test.dart

```diff
@@ -215,6 +215,21 @@ void main() {
         equals('ԪElement.updateChildԪ'),
       );
     });
+
+    test('getTrimmedUri', () {
+      expect(
+        getTrimmedUri('http://127.0.0.1:60667/72K34Xmq0X0=/#/vm').toString(),
+        equals('http://127.0.0.1:60667/72K34Xmq0X0=/'),
+      );
+      expect(
+        getTrimmedUri('http://127.0.0.1:60667/72K34Xmq0X0=').toString(),
```

Comment:
shouldn't we fix this particular case up to add a / so the url matches what the VM Service supports?

Reply:
done.